code/Base/GamemodeSystem/Entities/GamemodeGenericEntity.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace Sports;
+
+/// <summary>
+/// Utility entity for generic gamemode purposes.
+/// This entity will be gathered automatically by its linked gamemode and can be used via its tags and generic information for meta purposes.
+/// </summary>
+[HammerEntity]
+[Icon( "transform" )]
+[Category( "Gamemode Meta" )]
+[Title( "Generic Gamemode Entity" )]
+[EditorModel( "models/editor/info_target.vmdl" )]
+public class GamemodeGenericEntity : GamemodeEntity
```

Comment:
What's the point/difference between this and GamemodeEntity?

Reply:
this is a hammer entity. Putting stuff like HammerEntity, EditorModel etc on a base type and changing them on an inheriting type broke shit regarding gamedata sadly. I might make an issue for it at some point

Reply:
Do you think we'll end up using this in the editor at all? It doesn't have really any functionality